extensions/amp-form/0.1/amp-form.js

```diff
@@ -578,18 +579,36 @@ export class AmpForm {
     // Set ourselves to the SUBMITTING State
     this.setState_(FormState.SUBMITTING);
 
+    //Promised to run before all async calls ; require extended timeout
```

Comment:
Comment is out of date (no extended timeout).

`// Promises to run before submit without timeout.`